### PR TITLE
Preempt comment from Deb on repetitive text

### PR DESCRIPTION
**Pull request by danvangeest**

### CODE REVIEW

**Comment by bwesterb** [comment]
With this change, it suggests that we define that type here instead of citing the definition.

**Comment by danvangeest** [comment]
Line 191 says "NOTE: The above syntax is from {{!RFC5912}}..." although that is still not explicit without following the reference.

Line 191 could be changed to "... The above definition if from...", or line 179 could be changed to "The AlgorithmIdentifier type is defined in {{!RFC5912}} as follows:"

**Comment by bwesterb** [comment]
Perfect.

File: draft-ietf-lamps-dilithium-certificates.md [hunk 1]
```diff
@@ -176,8 +176,7 @@ to hash *mu* and the private key.
 
 # Identifiers {#oids}
 
-The AlgorithmIdentifier type, which is included herein for convenience,
-is defined as follows:
+The AlgorithmIdentifier type is defined as follows:
```
Comment (danvangeest) [inline draft-ietf-lamps-dilithium-certificates.md:179]:
```suggestion
The AlgorithmIdentifier type is defined in {{!RFC5912}} as follows:
```

**Comment by danvangeest** [comment]
> Perfect.

You didn't express a preference for either of the options I proposed so I picked one :)

**Pull request merged**